incuna-sass/functions/_functions-variables.sass

```diff
@@ -0,0 +1,5 @@
+// Set a default font-base variable here that can
+// be overwritten locally
+// This way we can import incuna-sass files above
+// locally defined variables without getting errors
+$font-base: 16px !default
```

Comment:
Yes, the intention of the `/functions` folder is that it contains specifically definitions of functions in sass (via `@function`)

Perhaps a top level `default.sass` or even `variables.sass` (to be consistent with projects)?